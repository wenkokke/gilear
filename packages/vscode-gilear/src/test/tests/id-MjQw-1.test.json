{
  "name": "id-MjQw",
  "file": "id-MjQw.gilear",
  "test": [
    {
      "edits": [
        {
          "range": {
            "start": { "character": 10, "line": 1 },
            "end": { "character": 11, "line": 1 }
          },
          "text": ""
        }
      ]
    },
    {
      "diagnostics": [
        {
          "severity": "error",
          "range": {
            "start": { "character": 10, "line": 1 },
            "end": { "character": 10, "line": 1 }
          },
          "message": "Syntax Error: Missing '';''"
        }
      ]
    },
    {
      "edits": [
        {
          "range": {
            "start": { "character": 10, "line": 1 },
            "end": { "character": 10, "line": 1 }
          },
          "text": ";"
        }
      ]
    },
    { "diagnostics": [] }
  ]
}
