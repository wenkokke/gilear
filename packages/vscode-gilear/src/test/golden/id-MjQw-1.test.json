{
  "file": "id-MjQw.gilear",
  "steps": [
    {
      "edits": [
        {
          "range": {
            "start": { "character": 10, "line": 1 },
            "end": { "character": 11, "line": 1 }
          },
          "text": ""
        }
      ]
    },
    {
      "diagnostics": [
        {
          "severity": "error",
          "range": {
            "start": { "character": 10, "line": 1 },
            "end": { "character": 10, "line": 1 }
          },
          "message": "Syntax Error: Missing '';''"
        }
      ]
    },
    {
      "edits": [
        {
          "range": {
            "start": { "character": 10, "line": 1 },
            "end": { "character": 10, "line": 1 }
          },
          "text": ";"
        }
      ]
    },
    { "diagnostics": [] }
  ]
}
